example/main.py

```diff
@@ -186,5 +189,14 @@ def _get_result_from_dialogflow(text_to_be_analyzed: str) -> str:
         raise
     return '\n'.join(fulfillment_message.text.text[0] for fulfillment_message in response.query_result.fulfillment_messages)
 
+def upload_message_to_bigquery(json : dict):
+    PROJECT_ID = 'pycontw-225217'
+    client = bigquery.Client(project=PROJECT_ID)
+    table = client.dataset("test").table("Uploadtest")
+    message = json['entry'][0]['messaging'][0]['message']['text']
+    timestamp = json['entry'][0]['messaging'][0]['timestamp']
+    fb_message = [{"timestamp":timestamp,"message":message}]
```

Comment:
Nice to have: if there are so many variables needed to be indexed by keys, maybe wrap them up with try-except statement could help the DAG to run more robustly and easier to debug (with logs).